tests/indexes/test_api.py

```diff
@@ -21,62 +21,138 @@
 OTUS_JSON_PATH = Path.cwd() / "tests/test_files/index/otus.json.gz"
 
 
-async def test_find(mocker, snapshot, fake2, spawn_client, static_time):
-    client = await spawn_client(authorize=True)
+class TestFind:
+    async def test(self, mocker, snapshot, fake2, spawn_client, static_time):
+        client = await spawn_client(authorize=True)
 
-    user = await fake2.users.create()
+        user = await fake2.users.create()
+
+        await client.db.indexes.insert_many(
+            [
+                {
+                    "_id": "bar",
+                    "version": 1,
+                    "created_at": static_time.datetime,
+                    "manifest": {"foo": 2},
+                    "ready": False,
+                    "has_files": True,
+                    "job": {"id": "bar"},
+                    "reference": {"id": "bar"},
+                    "user": {"id": user.id},
+                    "sequence_otu_map": {"foo": "bar_otu"},
+                },
+                {
+                    "_id": "foo",
+                    "version": 0,
+                    "created_at": static_time.datetime,
+                    "manifest": {"foo": 2},
+                    "ready": False,
+                    "has_files": True,
+                    "job": {"id": "foo"},
+                    "reference": {"id": "foo"},
+                    "user": {"id": user.id},
+                    "sequence_otu_map": {"foo": "foo_otu"},
+                },
+            ]
```

Comment:
This is not a very effective test as it doesn't check that both ready and non-ready indexes are returned for a plain `GET /indexes`.